dojson/contrib/to_marc21/fields/ad84188x.py

```diff
@@ -0,0 +1,263 @@
+            value.get('same_as_associated_field_b')
+        ),
+        'e': utils.reverse_force_list(
+            value.get('same_as_associated_field_e')
+        ),
+        'd': utils.reverse_force_list(
+            value.get('same_as_associated_field_d')
+        ),
+        'g': utils.reverse_force_list(
+            value.get('same_as_associated_field_g')
+        ),
+        'f': utils.reverse_force_list(
+            value.get('same_as_associated_field_f')
+        ),
+        'i': utils.reverse_force_list(
+            value.get('same_as_associated_field_i')
+        ),
+        'h': utils.reverse_force_list(
+            value.get('same_as_associated_field_h')
+        ),
+        'k': utils.reverse_force_list(
+            value.get('same_as_associated_field_k')
+        ),
+        'j': utils.reverse_force_list(
+            value.get('same_as_associated_field_j')
+        ),
+        'm': utils.reverse_force_list(
+            value.get('same_as_associated_field_m')
+        ),
+        'l': utils.reverse_force_list(
+            value.get('same_as_associated_field_l')
+        ),
+        'o': utils.reverse_force_list(
+            value.get('same_as_associated_field_o')
+        ),
+        'n': utils.reverse_force_list(
+            value.get('same_as_associated_field_n')
+        ),
+        'q': utils.reverse_force_list(
+            value.get('same_as_associated_field_q')
+        ),
+        'p': utils.reverse_force_list(
+            value.get('same_as_associated_field_p')
+        ),
+        's': utils.reverse_force_list(
+            value.get('same_as_associated_field_s')
+        ),
+        'r': utils.reverse_force_list(
+            value.get('same_as_associated_field_r')
+        ),
+        'u': utils.reverse_force_list(
+            value.get('same_as_associated_field_u')
+        ),
+        't': utils.reverse_force_list(
+            value.get('same_as_associated_field_t')
+        ),
+        'w': utils.reverse_force_list(
+            value.get('same_as_associated_field_w')
+        ),
+        'v': utils.reverse_force_list(
+            value.get('same_as_associated_field_v')
+        ),
+        'y': utils.reverse_force_list(
+            value.get('same_as_associated_field_y')
+        ),
+        'x': utils.reverse_force_list(
+            value.get('same_as_associated_field_x')
+        ),
+        'z': utils.reverse_force_list(
+            value.get('same_as_associated_field_z')
+        ),
+        '$ind1': value.get("ind1", "_"),
+        '$ind2': value.get("ind2", "_"),
+    }
+
+
+@to_marc21.over('883', '^machine_generated_metadata_provenance$')
+@utils.reverse_for_each_value
+@utils.filter_values
+def reverse_machine_generated_metadata_provenance(self, key, value):
+    """Reverse - Machine-generated Metadata Provenance."""
+    indicator_map1 = {"Fully machine-generated": "0", "No information provided/not applicable": "_", "Partially machine-generated": "1"}
```

Comment:
please try `autopep8 -i -aa ...` (`pip install autopep8`) and `s/"/'/` + `s/'''/"""/`.